Cargo.toml

```diff
@@ -362,6 +364,14 @@ crate-type = ["cdylib"]
 name = "android"
 path = "examples/android/android.rs"
 
+[[example]]
+name = "run_animation"
+path = "examples/animations/run_animation.rs"
+
+[[example]]
+name = "bench_run_idle_animation"
+path = "examples/animations/bench_run_idle_animation.rs"
```

Comment:
The examples also need to be added to the README.md in the examples folder.

Also, the `bench_run_idle_animation` doesn't exist (anymore) in this PR.

Reply:
Done